segmentation/scripts/transforms.py

```diff
@@ -1,17 +1,55 @@
 import albumentations as A
 import cv2
 
-def get_train_transform():
-    return A.Compose([
+
+def get_train_transform(use_color_transforms=False):
+    """
+    Returns a composition of data augmentations to be applied to images during training.
+
+    - Random crop to 256x256 size.
+    - Randomly applies one of the following augmentations with 75% probability:
+        - Horizontal Flip
+        - Vertical Flip
+        - 90-degree rotation.
+    - Optionally adds color transformations:
+        - RandomBrightnessContrast
+        - HueSaturationValue
+        - RandomGamma
+
+    Args:
+        use_color_transforms (bool, optional): If True, includes color transformations. Defaults to False.
+
+    Returns:
+        A.Compose: A pipeline of augmentations to be applied to training images.
+    """
+    transforms = [
         A.RandomCrop(height=256, width=256, always_apply=True),
         A.OneOf([
             A.HorizontalFlip(p=1),
             A.VerticalFlip(p=1),
             A.RandomRotate90(p=1),
```

Comment:
I think RandomScale with small range of possible ratio values will be beneficial.